calblueprint/sundayfriends
# Albert edit admin

Added functionality to edit admin information

## Testing

1. Ensure that edited information is correctly reflected in the frontend.
2. Ensure that edited information is correctly reflected in firebase. 
3. Ensure that incorrect edited information (i.e. not well-formed email, phone #) is unable to be saved.
4. Ensure that 'Cancel' functionality works as expected.

NOTE: there is a slight styling issue where 'Save' and 'Cancel' buttons are not correctly aligned; I will resolve in a future PR.

## Screenshots
<img width="1394" alt="Screen Shot 2022-01-01 at 9 38 25 PM" src="https://user-images.githubusercontent.com/25868533/147864698-4c5a3c
<img width="1388" alt="Screen Shot 2022-01-01 at 9 39 34 PM" src="https://user-images.githubusercontent.com/25868533/147864699-f00b1f12-4151-43b5-a26c-a7030f731277.png">
42-46e8-411d-ba07-5c875ccb54ca.png">
<img width="1389" alt="Screen Shot 2022-01-01 at 9 40 17 PM" src="https://user-images.githubusercontent.com/25868533/147864704-85838b40-2ee1-42f2-ac28-72b025cdf28b.png">

CC: @jacobjk01 @hukellyy
